From e8cddd39da8d691a4075d91996de1ab04fa1664e Mon Sep 17 00:00:00 2001
From: Pino Toscano <ptoscano@redhat.com>
Date: Tue, 11 Feb 2014 10:40:31 +0100
Subject: [PATCH] builder: isolate C libraries in an own OCAMLCLIBS

Just moving stuff within Makefile.am, no functional changes.

(cherry picked from commit c66cec6ffac2af400487b7fb4ff437eddc6e7aa7)
---
 builder/Makefile.am | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/builder/Makefile.am b/builder/Makefile.am
index 2be495b..78a9e72 100644
--- a/builder/Makefile.am
+++ b/builder/Makefile.am
@@ -113,10 +113,15 @@ endif
 OCAMLCFLAGS = -g -warn-error CDEFLMPSUVYZX $(OCAMLPACKAGES)
 OCAMLOPTFLAGS = $(OCAMLCFLAGS)
 
+OCAMLCLIBS  = \
+	$(LIBLZMA_LIBS) \
+	-pthread -lpthread \
+	-lncurses -lcrypt
+
 virt-builder: $(OBJECTS)
 	$(OCAMLFIND) ocamlopt $(OCAMLOPTFLAGS) \
 	  mlguestfs.cmxa -linkpkg $^ \
-	  -cclib '-pthread $(LIBLZMA_LIBS) -lncurses -lcrypt -lpthread' \
+	  -cclib '$(OCAMLCLIBS)' \
 	  $(OCAML_GCOV_LDFLAGS) \
 	  -o $@
 
-- 
1.8.5.3

